.changeset/blue-mangos-speak.md

```diff
@@ -0,0 +1,16 @@
+---
+'@commercetools-test-data/custom-view': minor
+'@commercetools-test-data/custom-view-installation': minor
+---
+
+New package to generate test data for Custom Views Installations.
+
+```ts
+import {
+  RestrictedCustomViewInstallationForOrganization,
+  type TRestrictedCustomViewInstallationForOrganizationGraphql,
+} from '@commercetools-test-data/custom-view-installation';
+
+const restrictedCustomViewInstallationForOrganization =
+  RestrictedCustomViewInstallationForOrganization.random().buildGraphql<TRestrictedCustomViewInstallationForOrganizationGraphql>();
+```
```

Comment:
I think we can keep one package `@commercetools-test-data/custom-view` and include the installation models in that as well.